<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BabibelWindow</class>
 <widget class="QMainWindow" name="BabibelWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Babibel</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="4" column="1" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="addContactButton">
        <property name="text">
         <string>Add Contact</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeContactButton">
        <property name="text">
         <string>Remove Contact</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="blockContactButton">
        <property name="text">
         <string>Block Contact</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="7" column="1" colspan="2">
     <widget class="QLabel" name="onlinePeople">
      <property name="text">
       <string>0 People Online</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="indent">
       <number>5</number>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="LOGO">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../ressources/images.qrc">:/Images/images/babibel.png</pixmap>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLineEdit" name="statusEdit">
        <property name="maxLength">
         <number>200</number>
        </property>
        <property name="placeholderText">
         <string>Status</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="statusComboBox">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Available</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Busy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Away</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Invisible</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="login">
        <property name="font">
         <font>
          <pointsize>17</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QListView" name="contactList">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="uniformItemSizes">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>statusEdit</tabstop>
  <tabstop>statusComboBox</tabstop>
  <tabstop>contactList</tabstop>
  <tabstop>addContactButton</tabstop>
  <tabstop>removeContactButton</tabstop>
  <tabstop>blockContactButton</tabstop>
 </tabstops>
 <resources>
  <include location="../ressources/images.qrc"/>
 </resources>
 <connections/>
</ui>
